app/views/service_providers/_form.html.erb

```diff
@@ -378,8 +378,26 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
       <% end %>
     </fieldset>
   <% else %>
-    <p class="usa-label">Help text</p>
-    <p>Do you need to edit the custom help text for your application? <a href="https://logingov.zendesk.com">Contact us</a>.</p>
+    <fieldset class="usa-fieldset-inputs custom-help-text">
+        <legend class="usa-label">Custom help text</legend>
+          <p>You can view your existing help text here.</p>
```

Comment:
[question] if they don't have any custom help text set, should we have a different view/text? like  `<p>You have no help text set </p>`. We should also maybe remove the view if of the help text if there is none set?

Reply:
Yeah thats a good idea - I will add that in